tests/testthat/test-qenv_eval_code.R

```diff
@@ -41,21 +41,21 @@ testthat::test_that("getting object from the package namespace works even if lib
 testthat::test_that("eval_code works with character", {
   q1 <- eval_code(qenv(), "a <- 1")
 
-  testthat::expect_identical(q1@code, "a <- 1")
```

Comment:
"Nature" of the `eval_code` and `@code` has changed and tests needs to be rewritten to fit better a current state. `@code` holds call asis, which means that many tests can be written like so:

```
# testthat {
  code <- "some code"
  q <- eval_code(<qenv>, code)
 # testthat::expect_identical(<drop_attributes>(q@code), as.list(code)) 
 # testthat::expect_identical(get_code(code), code)
}

```

Reply:
Now tests are written in this format

```r
testthat::expect_identical(get_code(code), paste(code, collapse = "\n"))
```

The

```
testthat::expect_identical(<drop_attributes>(q@code), as.list(code)) 
```

part from the suggestion is not true, as `code` can have multiple calls in one element and can have length 1, where `q@code` will have length greater than one for input with multiple calls.